tests/conftest.py

```diff
@@ -120,7 +120,7 @@ def _q_series(values, start="1/1/2000"):
             coords=[coords],
             dims="time",
             name="q",
-            attrs={"standard_name": "dis", "units": "m3 s-1"},
+            attrs={"standard_name": "streamflow", "units": "m3 s-1"},
```

Comment:
I'm not sure this is the standard. I think it's `water_volume_transport_in_river_channel`. To be confirmed.